app/styles/_avatars.scss

```diff
@@ -63,13 +63,13 @@ a.avatar-wrapper {
   color: transparent;
 }
 
-#fxa-settings-sub-header {
-  font-size: $medium-font;
-  font-weight: normal;
-  padding-bottom: 5px;
+#settings-home {
+  margin: 40px 0px -30px;
```

Comment:
You can drop the "px" units on 0px.
Not sure if you want to sort these A-Z (same with the new rules below).